Source/Plugin.BLE/Shared/Contracts/IBluetoothLE.cs

```diff
@@ -29,6 +30,13 @@ public interface IBluetoothLE
         /// </summary>
         bool IsOn { get; }
 
+        /// <summary>
+        /// Try set the state of the Bluetooth on/off
+        /// </summary>
+        /// <param name="on"></param>
+        /// <returns>true if the the method executed with success otherwice false</returns>
+        Task<bool> TrySetStateAsync(bool on);
```

Comment:
I think it would be a good idea to document here that this is only implemented on Windows (for the time being).